CHANGES.md

```diff
@@ -5,8 +5,10 @@ Change Log
 
 * Fixed a bug where a Model's compressed textures were not being displayed. [#5596](https://github.com/AnalyticalGraphicsInc/cesium/pull/5596)
 * Fixed a bug where jsep was undefined when using webpack [#5593](https://github.com/AnalyticalGraphicsInc/cesium/issues/5593)
+* Make Entity mode matrix public via computeModelMatrix(). [#5584](https://github.com/AnalyticalGraphicsInc/cesium/pull/5584)
```

Comment:
We usually say something like `Added Entity.computeModelMatrix which ...`.  End users have no idea what was private/public so you need to explain what the function is.